apps/system/test/marionette/lib/system.js

```diff
@@ -90,10 +94,18 @@ System.prototype = {
   client: null,
 
   URL: System.URL,
+  Keys: System.Keys,
   origin: System.URL.substring(0, System.URL.indexOf('/manifest.webapp')),
 
   Selector: System.Selector,
 
+  sendKeyToElement: function(element, key) {
```

Comment:
Since the `sendKeyToElement` method is moved here, should we remove the duplicate code in "lib/app_modal_dialog.js" [1]?

[1] https://github.com/mozilla-b2g/gaia/blob/master/tv_apps/smart-system/test/marionette/lib/app_modal_dialog.js#L107